lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -456,4 +472,53 @@ class IdentityCreatorController extends BaseController {
       );
     }
   }
+
+  void pickImage(BuildContext context, {int? maxWith}) async {
+    clearFocusEditor(context);
+
+    final filePickerResult = await FilePicker.platform.pickFiles(
+      type: FileType.image,
+      withData: true
+    );
+
+    if (context.mounted) {
+      final platformFile = filePickerResult?.files.single;
+      if (platformFile != null) {
+        _insertInlineImage(context, platformFile, maxWith: maxWith);
+      } else {
+        _appToast.showToastErrorMessage(
+          context,
+          AppLocalizations.of(context).cannotSelectThisImage
+        );
+      }
+    } else {
+      logError("IdentityCreatorController::pickImage: context is unmounted");
+    }
+  }
+
+  bool _isExceedMaxSizeInlineImage(int fileSize) =>
+    fileSize > IdentityCreatorConstants.maxSizeIdentityInlineImage.kiloByteToBytes;
+
+  void _insertInlineImage(
+    BuildContext context,
+    PlatformFile platformFile,
+    {int? maxWith}
+  ) {
+    if (_isExceedMaxSizeInlineImage(platformFile.size)) {
+      _appToast.showToastErrorMessage(
+        context,
+        AppLocalizations.of(context).pleaseChooseAnImageSizeCorrectly(
+          IdentityCreatorConstants.maxSizeIdentityInlineImage
+        )
+      );
+    } else {
+      if (PlatformInfo.isWeb) {
+        richTextWebController.insertImageAsBase64(platformFile: platformFile);
```

Comment:
you use the same method for Web and mobile platform

Reply:
No, Mobile and web use 2 different editors